lib/Optimizer/Transforms/DecompositionPatterns.cpp

```diff
@@ -110,6 +116,18 @@ class QuakeOperatorCreator {
     op->replaceAllUsesWith(newWireValues);
   }
 
+  void selectWiresAndReplaceUses(Operation *op, Value measOut,
+                                 ValueRange newValues) {
+    SmallVector<Value, 4> newWireValues;
+    newWireValues.push_back(measOut);
+    for (const auto &v : newValues)
+      if (v.getType().isa<quake::WireType>())
```

Comment:
type().isa<>() is deprecated, please use

 `if(mlir::isa<quake::WireType>(v.getType()))`